x-pack/plugins/observability_solution/infra/public/components/asset_details/tabs/dashboards/actions/edit_dashboard.tsx

```diff
@@ -17,27 +17,45 @@ export function EditDashboard({
   onRefresh,
   currentDashboard,
   assetType,
+  canLinkOrEditCustomDashboard,
 }: {
   onRefresh: () => void;
   currentDashboard: DashboardItemWithTitle;
   assetType: InfraCustomDashboardAssetType;
+  canLinkOrEditCustomDashboard: boolean;
 }) {
   const [isModalVisible, setIsModalVisible] = useState(false);
   const onClick = useCallback(() => setIsModalVisible(!isModalVisible), [isModalVisible]);
 
   return (
     <>
-      <EuiButtonEmpty
-        color="text"
-        size="s"
-        iconType="pencil"
-        data-test-subj="infraEditCustomDashboardMenu"
-        onClick={onClick}
+      <EuiToolTip
+        position="top"
+        content={
+          !canLinkOrEditCustomDashboard
+            ? i18n.translate(
+                'xpack.infra.linkDashboard.tooltip.youDoNotHavePermissionToUseThisFeature',
+                {
+                  defaultMessage:
+                    'You do not have permission to use this feature. Please ask your administrator for access.',
```

Comment:
The message assumes the user or the admin are aware that in order to link/add/delete dashboard they require saved object permissions, which may not be the case. 

Probably someone from docs could help us with the phrasing here.

Reply:
Good catch, thank you! It's a good idea to mention what permissions are needed 👍 
@roshan-elastic any ideas on how to improve the message, I can maybe find something from the docs team as well as @kpatticha suggested if you think we need help.